guides/common/modules/con_bootloader-binary-location-overview.adoc

```diff
@@ -0,0 +1,34 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="con_bootloader-binary-location-overview_{context}"]
+= Boot loader management and file structure
+
+This section describes how {Project} manages and serves boot loader binaries through the *{SmartProxy}*, which provides the underlying network boot services used during automated host provisioning.
+While traditional *PXE (Preboot Execution Environment)* remains a core method, {Project} also supports *HTTPBoot*, *iPXE*, and *UEFI HTTP(S) Boot* mechanisms.
+
+During a network-based installation, {SmartProxy} coordinates multiple services:
+
+* DHCP: Assigns IP addresses and provides the boot loader filename and `next-server` information.
+
+* TFTP: Delivers boot loaders such as PXELinux, GRUB, or iPXE binaries to clients.
+
+* HTTP(S): Serves boot loaders and installation images via the {SmartProxy}’s HTTPBoot module
+
+When a host begins a network boot, it retrieves the appropriate boot loader (for example, `pxelinux.0`, `grubx64.efi`, or `ipxe.efi`) according to its firmware type and *PXE Loader* setting in {Project}.
+The boot loader then loads {Project}-generated configuration files that define which kernel and initrd to boot, initiating the OS installation process.
+
+The *{SmartProxy}* manages the distribution of these boot loader binaries and configuration files.
+All boot loaders, whether used for PXE, iPXE, or HTTPBoot, are stored under the {SmartProxy}’s TFTP root directory (`/var/lib/tftpboot/`), which is also shared by the HTTPBoot service.
```

Comment:
UEFI HTTP Boot and Boot Loader HTTP Boot (planned here: https://github.com/theforeman/foreman-documentation/pull/4304/files#diff-9829fb8de26ac082d3a18e534b1aa4c2ae0d3cb4eae845a37484e52e207a77aaR6)

Here its UEFI HTTP Boot, not HTTPBoot

HTTPBoot = loading installation files via http
UEFI HTTP Boot = loading boot loadders